internal/http_api/api_request.go

```diff
@@ -38,6 +37,7 @@ func NewDeadlineTransport(timeout time.Duration) *http.Transport {
 			}
 			return &deadlinedConn{timeout, c}, nil
 		},
+		ResponseHeaderTimeout: requestTimeout,
```

Comment:
I don't _think_ we need to set this, considering we're bounding the _entire_ request by the same timeout.